common-npm-packages/azure-arm-rest-v2/package.json

```diff
@@ -13,14 +13,14 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "@azure/msal-node": "1.14.5",
+    "@azure/msal-node": "^1.16.0",
```

Comment:
We've pinned that version to `1.14.5` due to Node10 deprecation.
#17674

Reply:
Pinned the version back, thanks!